app/register-your-interest/routes/eligibility.js

```diff
@@ -0,0 +1,31 @@
+const Joi = require('joi')
+const Boom = require('@hapi/boom')
```

Comment:
should we put this under the `app/auto-eligibility folder`?

Reply:
done